source/features/unfinished-comments.tsx

```diff
@@ -9,18 +10,19 @@ function hasDraftComments(): boolean {
 	return select.all('textarea').some(textarea => textarea.value.length > 0 && textarea.offsetWidth > 0);
 }
 
-async function updateDocumentTitle(): Promise<void> {
-	if (documentTitle) {
-		document.title = documentTitle;
-		documentTitle = undefined;
-	} else if (document.visibilityState === 'hidden' && hasDraftComments()) {
+function updateDocumentTitle(): void {
+	if (document.visibilityState === 'hidden' && hasDraftComments() && !documentTitle) {
```

Comment:
I know it's ugly, but the nested `if` was here to account for the case where a comment is sent from a hidden tab but there are still drafts somewhere on the page (and thus the title shouldn't be reset).
But maybe this possibility doesn't exist in the first place?

Reply:
You're right, the 2 conditions aren't equivalent when "is hidden, has draft and has a documentTitle". I don't know if it matters at the moment because the change doesn't fix the issue, as mentioned in the other comment.